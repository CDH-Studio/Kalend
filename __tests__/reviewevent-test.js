import React from 'react';
import renderer from 'react-test-renderer';
// import ReviewEvent from '../src/components/screens/ReviewEvent';
import {StatusBar, StyleSheet} from 'react-native';
import {Header} from 'react-navigation';
import { FAB } from 'react-native-paper';
// import {HEIGHT} from '../src/components/TutorialStatus';

// test("ReviewEvent renders correctly", () => {
//     const tree = renderer.create(<ReviewEvent />).toJSON();
// 	expect(tree).toMatchSnapshot();
// });

test("StatusBar renders correctly", () => {
	const statusBar = renderer.create(<StatusBar translucent={true} backgroundColor={'#105dba'}/>).toJSON();
	expect(statusBar).toMatchSnapshot();
});

test("FAB renders correclty", () => {
    const fab = renderer.create(<FAB style={styles.fab} icon="check"/>).toJSON();
    expect(fab).toMatchSnapshot();
});

// const tutorialHeight = HEIGHT;
// const headerHeight = Header.HEIGHT;
// const containerHeight = containerHeight;

const styles = StyleSheet.create({
	container: {
		flex: 1
	},

	scrollView: {
		flex: 1,
		// marginTop: StatusBar.currentHeight + headerHeight,
		paddingHorizontal: 25
	},

	content: {
		flex:1,
		justifyContent:'space-evenly',
		// height: containerHeight,
		// paddingBottom: tutorialHeight + 16
	},

	sectionTitle: {
		color: '#565454',
		fontFamily: 'Raleway-SemiBold',
		fontSize: 20,
		marginTop: 20,
		marginBottom: 5
	},

	fab: {
		position: 'absolute',
		margin: 16,
		right: 0,
		bottom: 0,
		zIndex: 1 //To make it go on top of the tutorialStatus
	},
});
