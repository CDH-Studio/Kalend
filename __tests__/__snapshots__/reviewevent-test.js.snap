// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAB renders correclty 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#03dac4",
      "borderRadius": 28,
      "bottom": 0,
      "elevation": 6,
      "margin": 16,
      "opacity": 1,
      "position": "absolute",
      "right": 0,
      "shadowColor": "#000000",
      "shadowOffset": Object {
        "height": 5,
        "width": 0,
      },
      "shadowOpacity": 0.24,
      "shadowRadius": 6,
      "transform": Array [
        Object {
          "scale": 1,
        },
      ],
      "zIndex": 1,
    }
  }
  visible={true}
>
  <View
    accessibilityRole="button"
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Array [
        Object {
          "overflow": "hidden",
        },
        Object {
          "borderRadius": 28,
        },
      ]
    }
  >
    <View
      pointerEvents="none"
      style={
        Array [
          Object {
            "alignItems": "center",
            "flexDirection": "row",
            "justifyContent": "center",
          },
          Object {
            "height": 56,
            "width": 56,
          },
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "alignItems": "center",
              "justifyContent": "center",
            },
            Object {
              "height": 24,
              "width": 24,
            },
          ]
        }
      >
        <View
          style={
            Object {
              "bottom": 0,
              "left": 0,
              "opacity": 1,
              "position": "absolute",
              "right": 0,
              "top": 0,
              "transform": Array [
                Object {
                  "rotate": "0deg",
                },
              ],
            }
          }
        >
          <Text
            accessibilityElementsHidden={true}
            allowFontScaling={false}
            importantForAccessibility="no-hide-descendants"
            pointerEvents="none"
            style={
              Array [
                Object {
                  "color": "rgba(0, 0, 0, .54)",
                  "fontSize": 24,
                },
                Array [
                  Object {
                    "transform": Array [
                      Object {
                        "scaleX": 1,
                      },
                    ],
                  },
                  Object {
                    "backgroundColor": "transparent",
                  },
                ],
                Object {
                  "fontFamily": "Material Icons",
                  "fontStyle": "normal",
                  "fontWeight": "normal",
                },
                Object {},
              ]
            }
          >
            î—Š
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;

exports[`StatusBar renders correctly 1`] = `null`;
